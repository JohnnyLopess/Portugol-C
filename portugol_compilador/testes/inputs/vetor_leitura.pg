programa {
    funcao inicio() {
        inteiro v[2]
        leia(v[0])
        leia(v[1])
        escreva(v[0])
        escreva(v[1])
    }
}
