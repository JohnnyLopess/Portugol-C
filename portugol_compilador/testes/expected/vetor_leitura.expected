#include <stdio.h>
#include <string.h>

int main() {
    int v[2];
    scanf("%d", &v[0]);
    scanf("%d", &v[1]);
    printf("%d\n", v[0]);
    printf("%d\n", v[1]);
    return 0;
}
